src/test/java/BaseTest.java

```diff
@@ -26,16 +32,22 @@ public static void chromeConfigs() {
 
     @BeforeClass
     @Parameters({"BaseURL"})
-    public void launchBrowser(String BaseURL) {
-        ChromeOptions chromeOptions = new ChromeOptions();
-        chromeOptions.addArguments("--disable-notifications");
-
-        driver = new ChromeDriver(chromeOptions);
+    public void launchBrowser(String BaseURL) throws MalformedURLException {
+       // ChromeOptions chromeOptions = new ChromeOptions();
+       // chromeOptions.addArguments("--disable-notifications");
+//System.setProperty("webdriver.gecko.driver", "geckodriver");
+//driver= new FirefoxDriver();
+       driver = pickBrowser(System.getProperty("browser"));
         driver.manage().window().maximize();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
         url = BaseURL;
         driver.get(url);
+
     }
+
+   // public WebDriver getDriver(){
+      //  return threadDriver.get();
+   // }
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.